src/app/guildmanager/guildmanager.module.ts

```diff
@@ -5,11 +5,17 @@ import { MaterialModule } from '../shared/material.module';
 import { GuildManagerAppComponent } from './guildmanagerapp.component';
 import { FormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
+import { ToolbarComponent } from './components/toolbar/toolbar.component';
+import { SidenavComponent } from './components/sidenav/sidenav.component';
+import { MainContentComponent } from './components/maincontent/maincontent.component';
 
 @NgModule({
   declarations: [
     GuildManagerAppComponent,
-    GuildsComponent
+    GuildsComponent,
+    MainContentComponent,
+    ToolbarComponent,
+    SidenavComponent
```

Comment:
Should this be SideNav instead of Sidenav?

Reply:
The tutorial I was following had Sidenav, but yeah it was  bothering me too...Either way, I ended up not using these components right now anyways D: